common/components/internal-header/_internal-header.scss

```diff
@@ -1,311 +1,182 @@
   }
 }
 
 .app-header__navigation {
-  @include govuk-responsive-margin(2, "bottom");
-  display: block;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  list-style: none;
   margin: 0;
   padding: 0;
-  list-style: none;
 
-  @include mq ($from: desktop) {
-    float: right;
+  @media (min-width: 40.0625em) {
+    flex-direction: row;
+    align-items: center;
   }
 }
 
-.js-enabled {
-  .app-header__menu-button {
-    display: block;
-    @include mq ($from: desktop) {
-      display: none;
-    }
-  }
-
-  .app-header__navigation {
-    display: none;
-    @include mq ($from: desktop) {
-      display: block;
-    }
-  }
-
-  .app-header__navigation--open {
-    display: block;
-  }
-}
-
-
-.app-header__navigation--end {
-  @include mq ($from: desktop) {
-    margin: 0;
-    padding: govuk-spacing(1) 0;
-    text-align: right;
-  }
-}
-
-.app-header__navigation--no-service-name {
-  padding-top: govuk-spacing(7);
-}
-
 .app-header__navigation-item {
-  padding: govuk-spacing(2) 0;
-  border-bottom: 1px solid $govuk-header-nav-item-border-color;
+  @include govuk-font($size: 19);
+  margin-bottom: govuk-spacing(1);
+  text-align: right;
 
-  @include mq ($from: desktop) {
-    display: inline-block;
-    margin-right: govuk-spacing(3);
-    padding: govuk-spacing(1) 0;
-    border: 0;
+  @media (min-width: 40.0625em) {
+    margin-right: govuk-spacing(4);
+    padding-right: govuk-spacing(4);
+    margin-bottom: 0;
+    border-right: 1px solid #fff;
   }
 
   a {
-    @include govuk-font($size: 16, $weight: bold);
-    white-space: nowrap;
+    display: inline-block;
   }
-}
-
-.app-header__navigation-item--active {
-  a {
-    &:link,
-    &:hover,
-    &:visited {
-      color: $govuk-header-link-active;
-    }
 
-    // When focussed, the text colour needs to be darker to ensure that colour
-    // contrast is still acceptable
-    &:focus {
-      color: $govuk-focus-text-colour;
-    }
+  &:last-child {
+    margin-right: 0;
+    border-right: 0;
+    padding-right: 0;
   }
 }
 
-.app-header__navigation-item:last-child {
-  margin-right: 0;
-}
-
-@include govuk-media-query($media-type: print) {
-  .app-header {
-    border-bottom-width: 0;
-    color: govuk-colour("black");
-    background: transparent;
-  }
-
-  // Hide the inverted crown when printing in browsers that don't support SVG.
-  .app-header__logotype-crest-fallback-image {
-    display: none;
-  }
-
-  .app-header__link {
-    &:link,
-    &:visited {
-      color: govuk-colour("black");
-    }
+.app-header__link--service-name {
+  height: 100%;
+  align-items: center;
+  display: flex;
+  margin-right: govuk-spacing(4);
+  padding-right: govuk-spacing(4);
+  border-right: 1px solid #fff;
 
-    // Do not append link href to GOV.UK link when printing (e.g. '(/)')
-    &:after {
-      display: none;
-    }
+  &:last-child {
+    margin-right: 0;
+    border-right: 0;
+    padding-right: 0;
   }
 }
-
-// Begin adjustments for font baseline offset
```

Comment:
I think it would also be good to keep these at the bottom of the file as overrides. They're things currently required to work with the font we have and these allow those fixes to stand out quite nicely.

Reply:
I believe I may have fixed the issue that required these styles to be necessary? I changed the SVG path and attrs to be the same as HMPPS DS and from what I can see, the text in the header and the crest are lined up perfectly? (same as HMPPS DS)

Reply:
Yes, that's true but it was more the approach if settings the adjustments at the end of the page. I think the same minus top position has just been moved inside that statement instead of noted as an adjustment.

It feels like we've strayed enough to maybe not warrant needing them defined like this so fine to leave it. The adjustments were to cater for not using GDS Transport (I think).